### PR TITLE
Added post about 1.1.0 release

### PR DESCRIPTION
**Pull request by tboerger**

### CODE REVIEW

**Comment by andreynering** [comment]
LGTM

**Comment by joubertredrat** [comment]
I can't vote but you have my support :+1:

File: content/post/release-of-1.1.0.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+date: "2017-03-09T20:15:00+01:00"
+author: "tboerger"
+title: "Release of 1.1.0"
+tags: ["release"]
+draft: false
+---
+
+We proudly present the release of Gitea version 1.1.0. We have closed [126](https://github.com/go-gitea/gitea/issues?q=is%3Aissue+is%3Aclosed+milestone%3A1.1.0) issues and we also merged [348](https://github.com/go-gitea/gitea/pulls?utf8=%E2%9C%93&q=is%3Apr%20is%3Amerged%20milestone%3A1.1.0) pull requests to release this version of Gitea. You can download one of our pre-built binaries from our [downloads page](https://dl.gitea.io/gitea/1.1.0/), you just need to select the correct platform. For further details of the installation follow our [installation guide](https://docs.gitea.io/en-us/install-from-binary/).
+
+<!--more-->
```
Comment (geek1011) [inline content/post/release-of-1.1.0.md]: You could highlight some of the more significant features and include screenshots.

Comment (tboerger) [inline content/post/release-of-1.1.0.md]: It's the same message as for the other releases.

**Comment by metalmatze** [comment]
LGTM, deploy it! I was already on the blog looking for that blog post. :smile:

File: content/post/release-of-1.1.0.md (hunk 1 above)
Comment (geek1011) [inline content/post/release-of-1.1.0.md]: @tboerger OK, then it's good.

**Pull request merged**